test/test_basic.py

```diff
@@ -107,7 +107,7 @@ def wait_status(self, sessionid, expected_status, iterations=10):
         else:
             self.fail(f'session status was not updated to {expected_status}, still at {status}')
 
-    def newSession(self):
+    def newSession(self, tag='latest'):
```

Comment:
Hmm what would latest mean here? centos9? Shouldn't we be explicit.

Reply:
You are right, done.